scripts/powershell/sub-cleanup/sub-cleanup.ps1

```diff
@@ -16,26 +16,20 @@ subscription.  Complient resource groups are named with a
 valid Microsoft alias or using the format: alias-<identifier>
 
 Usage:
-    sub-cleanup.ps1 <subscription Id> <exception file>
+    sub-cleanup.ps1 <subscription Id>
```

Comment:
Nit: not really worth changing now, but in general you can use built-in powershell methods for doing this, so it's more consistent with other powershell experiences. See the New-TEstResources.ps1 script for how to use `[CmdletBinding()]` and comment-based help, that then works with the built-in `help` cmdlet (technically, that's an alias for `get-help`) or `-?` lone parameter.